lib/src/modules/conditional.dart

```diff
@@ -316,14 +318,24 @@ class Sequential extends _Always {
           // driving it, so hold onto it for later
           _driverInputsPendingPostUpdate.add(driverInput);
           if (!_pendingPostUpdate) {
-            unawaited(Simulator.postTick.first.then((value) {
-              // once the tick has completed, we can update the override maps
-              for (final driverInput in _driverInputsPendingPostUpdate) {
-                _inputToPreTickInputValuesMap[driverInput] = driverInput.value;
-              }
-              _driverInputsPendingPostUpdate.clear();
-              _pendingPostUpdate = false;
-            }));
+            unawaited(
+              Simulator.postTick.first.then(
+                (value) {
+                  // once the tick has completed,
+                  // we can update the override maps
+                  for (final driverInput in _driverInputsPendingPostUpdate) {
+                    _inputToPreTickInputValuesMap[driverInput] =
+                        driverInput.value;
+                  }
+                  _driverInputsPendingPostUpdate.clear();
+                  _pendingPostUpdate = false;
+                },
+              ).catchError((dynamic err, dynamic stackTrace) {
```

Comment:
From the docs:
```
  // The `Function` below stands for one of two types:
  // - (dynamic) -> FutureOr<T>
  // - (dynamic, StackTrace) -> FutureOr<T>
```

It also looks like you can use the `test` argument to check whether it is an exception so you don't need the if statement. 
`test: (Object err) => err is Exception`

That means I think the signature should be:
`(Exception err, StackTrace stackTrace)`